core/src/main/kotlin/com/cognifide/apmt/actions/Aem.kt

```diff
@@ -0,0 +1,6 @@
+package com.cognifide.apmt.actions
+
+object Aem {
+    const val CSRF_ENDPOINT: String = "/libs/granite/csrf/token.json"
```

Comment:
You don't need to put constants into object. The may be defined directly in file.